crates/sozo/ops/src/migrate/mod.rs

```diff
@@ -97,6 +99,8 @@ where
         let resources_have_changed =
             if !self.diff.is_synced() { self.sync_resources(ui).await? } else { false };
 
+        let external_contracts_have_changed = self.sync_external_contracts(ui).await?;
+
```

Comment:
For now that this task is totally independent, let move it before the world or after the initialization of the contracts.

This may change later if we implement the registration of the tokens, since registering a token requires the permissions to be set. (So it may be placed between permissions and initialization of contracts (which may use a token during this phase, but this is for later)